keras_contrib/applications/densenet.py

```diff
@@ -382,10 +382,10 @@ def __dense_block(x, nb_layers, nb_filter, growth_rate, bottleneck=False, dropou
     x_list = [x]
 
     for i in range(nb_layers):
-        x = __conv_block(x, growth_rate, bottleneck, dropout_rate, weight_decay)
-        x_list.append(x)
+        conv_block = __conv_block(x, growth_rate, bottleneck, dropout_rate, weight_decay)
+        x_list.append(conv_block)
 
-        x = concatenate(x_list, axis=concat_axis)
+        x = concatenate([x, conv_block], axis=concat_axis)
```

Comment:
My version is ever so slightly different from both of your versions. I think my version may be slightly more accurate w.r.t. the paper's description. That said, it is also possible that this change of yours could also outperform the original paper. :-)

I looked carefully at the following before writing the code above:
- your previous version
- your current version
- the paper and its [corresponding code](https://github.com/SimJeg/FC-DenseNet/blob/master/FC-DenseNet.py#L67)

Note that in the version I posted `x_list` and `return_concat_list` are gone. I may very well still be mistaken here, but could you either correct me or confirm my suspicion?